src/py/_pyodide/_core_docs.py

```diff
@@ -1027,10 +1027,14 @@ def children(self) -> Sequence["JsDomElement"]:
     def appendChild(self, child: "JsDomElement") -> None:
         pass
 
-    def addEventListener(self, event: str, listener: Callable[[Any], None]) -> None:
+    def addEventListener(
+        self, event: str, listener: Callable[[Any], None], kwargs: Any
+    ) -> None:
         pass
 
-    def removeEventListener(self, event: str, listener: Callable[[Any], None]) -> None:
+    def removeEventListener(
+        self, event: str, listener: Callable[[Any], None], kwargs: Any
```

Comment:
```suggestion
        self, event: str, listener: Callable[[Any], None], **kwargs: Any
```

Reply:
I think this does actually want to be `kwargs` (or something like `options`) if we want to pass along a single object with all the kwargs passed as attributes, yes? (Duplicate of above)